### PR TITLE
chore: update hello-world-bot to v3

### PR DESCRIPTION
**Pull request by yukun-dong**

### CODE REVIEW

File: hello-world-bot/teamsfx/app.local.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+version: 1.0.0
+
+registerApp:
+  - uses: teamsApp/create # Creates a Teams app
+    with:
+      name: bot # Teams app name
```
Comment (hund030) [inline hello-world-bot/teamsfx/app.local.yml:6]: Is this name expected?

File: hello-world-bot/.gitignore [hunk 2]
```diff
@@ -1,10 +1,10 @@
-# TeamsFx files
-node_modules
-.fx/configs/localSettings.json
-.fx/states/*.userdata
+# dependencies
+node_modules/
+
+# misc
+.env
```
Comment (hund030) [inline hello-world-bot/.gitignore:5]: I think we need to ignore .env.local file. Is this file copied from template?

Comment (yukun-dong) [reply hello-world-bot/.gitignore:5]: Yes, I will update template as well.

File: hello-world-bot/teamsfx/app.local.yml (hunk 1 above)
Comment (yukun-dong) [reply hello-world-bot/teamsfx/app.local.yml:6]: Good catch. I will rename it to hello-world-bot.

**Review by hund030 — APPROVED** [review]

**Pull request merged**